src/lavatory/policies/default.py

```diff
@@ -2,5 +2,13 @@
 
 
 def purgelist(artifactory):
+    """Default policy.
```

Comment:
I had this as a one line doc string on purpose. I think printing out the doc string is a good way to get a description of each plugin but if it is multi-line like this it does not look as good. 

I was going to make part of the plugin/policy spec be a doc string describing what the policy does so we can auto-generate a wiki page of the policies.

Reply:
Will remove this one then.